### PR TITLE
Use `waitress` prod server instead of `flask` development server

### PR DESCRIPTION
**Pull request by abnegate**

### CODE REVIEW

File: runtimes/python-3.10/requirements.txt [hunk 1]
```diff
@@ -1 +1,2 @@
-Flask==2.0.1
\ No newline at end of file
+Flask==2.0.1
```
Comment (Meldiron) [inline runtimes/python-3.10/requirements.txt:1]: We still kept Flask dependency

**Review by Meldiron — CHANGES_REQUESTED** [review]
When ready, please apply all changes to all versions of Python runtime.

**Comment by eldadfux** [comment]
Tests are failing

File: runtimes/python-3.10/requirements.txt (hunk 1 above)
Comment (abnegate) [reply runtimes/python-3.10/requirements.txt:1]: Yep Flask is still providing the framework, we are just replacing the web server

**Review by eldadfux — APPROVED** [review]

**Review by Meldiron — APPROVED** [review]
LGTM

**Pull request merged**